configs/dev-config.yaml

```diff
@@ -2,7 +2,7 @@
 tornado:
   port: 8888
   debug: true # will reload server on each change to python files
-  xsrf: true # need to make sure frontend is set up to work properly with this setting on
+  xsrf: false # can keep this false for dev mode
```

Comment:
Any reason we need to change this? I think frontend is working with xsrf tokens now?

Reply:
Yeah that's why I am changing it to false by default, when testing with FE auth stuff, this can be set to true. I am changing it to false because for Postman, it requires too many extra unnecessary steps.

Reply:
So is the plan to deal with that issue in the future? Like change this back to xsrf: true and deal with those extra unnecessary steps?

Reply:
Not sure what you mean by deal with the issue. It’s not an issue, it is what is expected when using Postman because it’s not a browser. Those extra steps will always be needed if using postman and having this set as true.

Reply:
Hmm I see. Maybe I'm misunderstanding, but I trust that this is good to go. Just wanted to ask!